app/zipper/api.go

```diff
@@ -68,18 +71,11 @@ func StartCarbonZipper(config cfg.Zipper,logger *zap.Logger){
 		}
 	}()
 
-	expvar.NewString("BuildVersion").Set(BuildVersion)
-	logger = zapwriter.Logger("main")
-	logger.Info("starting carbonzipper",
-		zap.String("build_version", BuildVersion),
-		zap.Any("config", config),
-	)
-
-	runtime.GOMAXPROCS(config.MaxProcs)
+	runtime.GOMAXPROCS(app.config.MaxProcs)
```

Comment:
this kind of runtime-wide configuration is better left in a main.go setting, I think. It's not super nice behavior for a library to modify this kind of thing behind your back.